cts/cli/regression.tools.exp

```diff
@@ -23,6 +23,481 @@ A new shadow instance was created. To begin using it, enter the following into y
+      * Possible values: time (default: )
+
+    * shutdown-escalation: Exit immediately if shutdown does not complete within this much time. If you need to adjust this value, it probably indicates the presence of a bug.
+      * Possible values: time (default: )
+
+    * join-integration-timeout: If you need to adjust this value, it probably indicates the presence of a bug.
+      * Possible values: time (default: )
+
+    * join-finalization-timeout: If you need to adjust this value, it probably indicates the presence of a bug.
+      * Possible values: time (default: )
+
+    * transition-delay: Enabling this option will slow down cluster recovery under all conditions
+      * Delay cluster recovery for this much time to allow for additional events to occur. Useful if your configuration is sensitive to the order in which ping updates arrive.
+      * Possible values: time (default: )
+
+    * stonith-enabled: Whether nodes may be fenced as part of recovery
+      * If false, unresponsive nodes are immediately assumed to be harmless, and resources that were active on them may be recovered elsewhere. This can result in a "split-brain" situation, potentially leading to data loss and/or service unavailability.
+      * Possible values: boolean (default: )
+
+    * startup-fencing: Whether to fence unseen nodes at start-up
+      * Setting this to false may lead to a "split-brain" situation, potentially leading to data loss and/or service unavailability.
+      * Possible values: boolean (default: )
+
+  * Deprecated options:
+
+    * remove-after-stop: Whether to remove stopped resources from the executor
+      * Values other than default are poorly tested and potentially dangerous. This option will be removed in a future release.
+      * Possible values: boolean (default: )
+=#=#=#= End test: List all available cluster options - OK (0) =#=#=#=
+* Passed: crm_attribute  - List all available cluster options
+=#=#=#= Begin test: List all available cluster options (XML) =#=#=#=
+<pacemaker-result api-version="X" request="crm_attribute --list-options=cluster --output-as=xml">
+  <resource-agent name="cluster-options" version="">
+    <version>1.1</version>
+    <longdesc lang="en">Also known as properties, these are options that affect behavior across the entire cluster. They are configured within cluster_property_set elements inside the crm_config subsection of the CIB configuration section.</longdesc>
+    <shortdesc lang="en">Pacemaker cluster options</shortdesc>
+    <parameters>
+      <parameter name="dc-version" advanced="false" generated="true">
+        <longdesc lang="en">Includes a hash which identifies the exact revision the code was built from. Used for diagnostic purposes.</longdesc>
+        <shortdesc lang="en">Pacemaker version on cluster node elected Designated Controller (DC)</shortdesc>
+        <content type="string" default=""/>
+      </parameter>
```

Comment:
@tomjelinek @gao-yan , and here's what the `--output=as=xml` output would look like. It's the same as the subdaemon metadata commands except it's wrapped in `<pacemaker-result>` (and it shows all the options).